content/research/presentations.md

```diff
@@ -0,0 +1,67 @@
+---
+title: "Presentations"
+date: 2021-05-28T09:44:45-05:00
+draft: false
+layout: page
+---
+
+<div class="presentation" >
+    <h2 style="text-align: center;">Posters</h2>
+    <div class="presentation_section">
+        <div class="info-box">
+            <p><strong>Isaac Smith</strong> presented a poster on “Documenting Scientific Software” at <i>Mid-Michigan Symposium for Undergraduate Research Experiences 2021</i> where he demonstrated how he made TARDIS documentation more user-friendly and interactive. </p>
+        </div>
+        <div class="preview-box">
+            <div class="preview-pdf" >
+                <iframe src="https://drive.google.com/file/d/1tF8QMSAXwheVJIz3WC9i-7EHRD7rfyA-/preview" width="100%" height="100%  allow="autoplay"></iframe>
+            </div>
+        </div>
+    </div>
+    <br>
+    <div class="presentation_section" >
+        <div class="preview-box">
+            <div class="preview-pdf" >
+                <iframe src="https://drive.google.com/file/d/15SNxFEQn--4D0QPLzKaXgJHVBBnoL9Dg/preview" width="100%" height="100%"allow="autoplay"></iframe>
+            </div>
+        </div>
+        <div class="info-box">
+            <p><strong>Ezequiel Passaro</strong> presented a poster on “An Open Source Framework for Atomic Data” for <i>Reunión anual de la Asociación Argentina de Astronomía</i> in September ’19 where he demonstrated the TARDIS package carsus.  </p>
+        </div>
+    </div>
+    <br>
```

Comment:
Can you please convert this html into Hugo shortcode - that will take 3 params: 
1. preview type: "pdf" or "video"
2. preview link
3. info text in markdown

See #18 for e.g.